generators/deployment/index.js

```diff
@@ -157,6 +158,15 @@ module.exports = class extends Generator {
 		this.cfIgnoreContent = ['.git/', 'node_modules/', 'test/', 'vcap-local.js'];
 	}
 
+	_configureGo() {
+		// Need a direct github link becasue 'go_buildpack' doesn't have dep support
+		this.manifestConfig.buildpack = 'https://github.com/cloudfoundry/go-buildpack.git#v1.8.25';
+		this.manifestConfig.command = undefined;
```

Comment:
there's no go command to specify here?

Reply:
https://docs.cloudfoundry.org/buildpacks/go/index.html#start_command

`If you do not specify a start command in a Procfile, in the manifest, or with the -c flag for cf push, the generated binary will be used as the start command.`